cmd/docker/main.go

```diff
@@ -42,22 +42,37 @@ import (
 )
 
 type CliOptions struct {
-	serverPort    uint
-	portLayerAddr string
-	proto         string
+	serverPort    *uint
+	portLayerAddr *string
+	portLayerPort *uint
+	debug         *bool
+
+	proto string
 }
 
 const (
 	productName    = "vSphere Integrated Containers"
 	clientHostName = "client.localhost"
 )
 
-var vchConfig config.VirtualContainerHostConfigSpec
+var (
+	vchConfig config.VirtualContainerHostConfigSpec
+	cli       CliOptions
+)
 
 func init() {
 	log.SetFormatter(viclog.NewTextFormatter())
 	trace.Logger.Level = log.DebugLevel
 	pprof.StartPprof("docker personality", pprof.DockerPort)
+
+	flag.Usage = Usage
```

Comment:
Is it really needs to be in inti() function? all parameter parsing work needs to be called from main() or any other deterministic place.